python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -194,26 +194,25 @@ def initialize_progress_bars(self, index: int, verbose_progress: bool) -> int:
         """Create progress bars at the given index (line offset in console).
 
         For AllToAllOperator, zero or more sub progress bar would be created.
-        Return the number of progress bars created for this operator.
+        Return the number of enabled progress bars created for this operator.
         """
         is_all_to_all = isinstance(self.op, AllToAllOperator)
         # Only show 1:1 ops when in verbose progress mode.
-        enabled = is_all_to_all or (
-            DataContext.get_current().enable_progress_bars and verbose_progress
+        progress_bar_enabled = DataContext.get_current().enable_progress_bars and (
+            is_all_to_all or verbose_progress
         )
         self.progress_bar = ProgressBar(
```

Comment:
ProgressBar is bit confusing name because it has responsibilities for in the monitoring lifecycle: if progress bar is not initialized the program do not work.

I'd suggest a change to ProgressMonitor - which can then have progress-bar property.

Reply:
Got it, I agree with you in general, I think it's a nit for now, so we can live with `ProgressBar` without updating too many places.